### ISSUE TITLE
Update `binder/environment.yml`

### ISSUE THREAD

**Issue by TomMonks** [health-data-science-OR/forecasting#5]
Works on Linux:

```
name: hds_forecast_2022
channels:
  - conda-forge
dependencies:
  - ipywidgets=7.6.5
  - jupyterlab=3.2.4
  - jupyterlab-plotly-extension=1.0.0
  - matplotlib=3.5.0
  - numpy=1.21.4
  - pandas=1.3.4
  - pip=21.0.1
  - plotly=5.4.0
  - python=3.8.8
  - scipy=1.7.3
  - seaborn=0.11.2
  - statsmodels=0.11.1
  - pip:
    - tensorflow==2.7.0
    - pmdarima==1.6.0
    - scikit-learn==0.23.0
    - pystan==2.19.1.1
    - prophet==1.0.1
    - forecast-tools==0.1.6
```

> Note: pytorch 1.10 was excluded as it resulted in conflicts.  May require separate environment.

**Comment by TomMonks** [comment]
updated in dev.

**Comment by TomMonks** [comment]
further updated to have pytorch 1.10 after testing on Pop!_OS

Needs to be tested on OpenStack

**Comment by TomMonks** [comment]
modified for tensorflow CPU only

```
name: hds_forecast_2022
channels:
  - conda-forge
dependencies:
  - ipywidgets=7.6.5
  - jupyterlab=3.2.4
  - jupyterlab-plotly-extension=1.0.0
  - matplotlib=3.5.0
  - numpy=1.21.4
  - pandas=1.3.4
  - pip=21.0.1
  - plotly=5.4.0
  - python=3.8.8
  - scipy=1.7.3
  - seaborn=0.11.2
  - statsmodels=0.11.1
  - pip:
    - tensorflow-cpu==2.7.0
    - pmdarima==1.6.0
    - scikit-learn==0.23.0
    - pystan==2.19.1.1
    - prophet==1.0.1
    - forecast-tools==0.1.6
```

**Comment by TomMonks** [comment]
merged v2.0.0
